contributors/devel/development.md

```diff
@@ -134,7 +134,9 @@ development environment, please [set one up](http://golang.org/doc/code.html).
 | 1.5, 1.6       | 1.7 - 1.7.5 |
 | 1.7            | 1.8.1       |
 | 1.8            | 1.8.3       |
-| 1.9+           | 1.9.1       |
+| 1.9            | 1.9.2       |
+| 1.10           | 1.9.3       |
```

Comment:
this is go1.9.1: https://github.com/kubernetes/kubernetes/blob/release-1.10/hack/lib/golang.sh#L329

Reply:
What's your opinion? For the 1.9 and 1.10 branches, only show the minimum go-lang version instead of the actual used of each branch?

Reply:
yes, it should be the minimum version for that branch.

Reply:
ok, I've modified and please check it.